### ISSUE TITLE
Default phase templates for 'Ask an Expert'

### ISSUE THREAD

**Issue by acshields** [appirio-tech/connect-app#2469]
@vikasrohit: There is a new special product offering with Azure that is getting initiated called 'Ask an Expert.'  The project intake form for this offering has already been created (https://connect.topcoder.com/new-project/azure).

The default (and only) phase template that should be displayed is Development (3 Milestones).

Aha! Link: https://topcoder.aha.io/features/TCCONNECT-440

**Comment by vikasrohit** [comment]
@acshields Yes, I created that template last week. It is hidden on catalog right now and only accessible via deep links which you shared above.

Right now it has the only phase `Development Iteration (1 report)`. So, if I understand it correctly, I need to change that to `Development Iteration (3 reports)`, correct?

And where do I need to put `Ask an Expert` thing?

**Comment by acshields** [comment]
Hi @vikasrohit - Actually, what you put as the default phase should be fine.

I am not sure about your question on where to put the "Ask an Expert" thing? Because this is a special offering it will be accessed by customers through this pathway: Azure Marketplace/Topcoder Offerings - Customer selects "ask an expert" offering - customer should be linked into Connect's Ask an Expert offering via the link above.  Currently, my understanding is that we will not be able to port over the customer's registration data so they would also need to complete the registration form on Connect as well.

**Comment by vikasrohit** [comment]
@acshields Okay.

IMO, `Ask an Expert` can not be a offering name unless we have it listed under a category e.g. at Azure Marketplace it is listed under Topcoder Offerings. So, if we want to list it as `Ask an Expert` on Connect, we need to have a category (say `Azure`) on the first screen just like we have for QA, Design and Software Development. And then under that category we can list `Ask an Expert` as an option. However, we can not that until we have more than one options in Azure category because our system takes assumes the only option in that category and takes user directly to the project creation form.

In short, I just need the place where I need to put the text `Ask an Expert` in our flow, if we need it at all.

**Comment by acshields** [comment]
@vikasrohit - Gotcha! Azure, as a category, will have several offerings associated with it: 
FREE - Ask the Azure Expert
Azure Technical Ideation Research 
Sharepoint Starter Pack
Azure Data Insights & Visualization
Azure Computer Vision PoC
Azure Blockchain Smart Contract PoC
Azure QA Test Cycles

However, the business is not ready to launch the additional offerings.  For setting up the Azure category, would it be possible to include these offerings but to **not** display these offerings and only display Ask an Expert until the business is ready to launch the remaining offerings?

**Comment by acshields** [comment]
@vikasrohit - Per our Slack conversation.  Got the final copy from the biz on the Ask the Expert.  See image/recap below.

![screen shot 2018-09-06 at 3 23 56 pm](https://user-images.githubusercontent.com/41118149/45188575-c4ad7b00-b1e9-11e8-8204-abd857885559.png)

Change Recap:
**1) Title Text:** Azure Ask the Expert
**2) Sub Title Text:** There is a lot you can do with Microsoft Azure and our experts are here to help you understand your options and make smart Azure-based decisions. Our Topcoders are excited to help you right now.
**3) REF Code Pre-Population:** AZUREX
**4) Question 1 Title Text:** What question would you like answered? Are you looking for a technical recommendation? Care to explain a scenario you’d want an expert opinion on? Tell us here:
**5) DELETE Post-Script under Question 1**
**6) Question 2 Additional Notes Sub Title Text:** What other circumstances, business requirements or restrictions, or other information should we know about that will help us answer your question most effectively?

**Comment by vikasrohit** [comment]
Done. We do not need **3** because we already have support for automatically fill/assume the ref code via URL e.g. `https://connect.topcoder.com/new-project/azure?refCode=AZUREX` will cause all projects created using this URL would have the correct ref code. (When refCode is present in the URL, we don't present user with option to edit that so that we are sure that the project is created using the intended ref code).
